release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -815,3 +834,20 @@ must also link to the build config object.
 the service catalog. The `controller-manager` and `apiserver` Pods show a status
 of `CrashLoopBackOff`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821589[*BZ#1821589*])
+
+* Installing {rh-openstack} 16 with Kuryr does not work due to the
+`ovn_controller` crashing after initial setup.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1812009[*BZ#1812009*])
+
+* The Red Hat Virtualization (RHV) machine `instance-state` annotation and the
+`providerStatus.instanceState` status do not always match. This mismatch causes
+the client to fail or incorrectly patch the RHV machine status.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1815394[*BZ#1815394*])
+
+* When scaling up a MachineSet on RHV, the new machine cannot exit the
+`Provisioned` phase.
```

Comment:
On skimming the bug, I'm not sure if describing the ramifications of this adds a lot at this point, but it's an option. Your call!